app/frontend/src/features/messages/SendField.tsx

```diff
@@ -30,15 +57,19 @@ export default function SendField({
   });
 
   return (
```

Comment:
I noticed when a message is sent, this input is not cleared. We should probably do that since that's quite standard chat behaviour.

Reply:
Wait is this fixed in the next PR in #513?

Reply:
yes